evalml/pipelines/components/transformers/encoders/target_encoder.py

```diff
@@ -65,9 +67,14 @@ def __init__(
             "category_encoders",
             error_msg="category_encoders not installed. Please install using `pip install category_encoders`",
         )
+        # Supress warnings for now due to problems discussion here:
+        # https://github.com/scikit-learn-contrib/category_encoders/issues/327
+        with warnings.catch_warnings():
```

Comment:
We could also change the default value of `smoothing` and add `min_samples_leaf` parameter but there is still some discussion as to what the new defaults should be so figured it was better to suppress the warnings

Reply:
agreed!